src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -472,13 +473,18 @@ export class PatientViewPageStore {
                             });
                         }
                     );
-                    return client.fetchGenericAssayDataInMultipleMolecularProfilesUsingPOST(
+                    const genericAssayRawData = await client.fetchGenericAssayDataInMultipleMolecularProfilesUsingPOST(
                         {
                             genericAssayDataMultipleStudyFilter: {
                                 sampleMolecularIdentifiers,
                             } as GenericAssayDataMultipleStudyFilter,
                         }
                     );
+                    if (
```

Comment:
is the case where we have invalid data something that should immediately be fixed?   i ask because we fail silently here

Reply:
Data fixing is on the way, it's not that urgent since this data is not public yet.